src/components/HistoryEntry.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+export default function HistoryEntry({ nameOfGame, players }) {
+  return (
+    <Wrapper>
+      {nameOfGame}
+      {players.map((player, index) => (
+        <Player key={index}>
```

Comment:
Maybe make sure player names are unique or create a unique id for each player on add?

Reply:
Yes, will do :)